opm/input/eclipse/Schedule/Well/WellKeywordHandlers.cpp

```diff
@@ -104,7 +104,7 @@ void handleWCONHIST(HandlerContext& handlerContext)
 
             auto table_nr = record.getItem("VFP_TABLE").get< int >(0);
             if (record.getItem("VFP_TABLE").defaultApplied(0)) {
-                table_nr = properties->VFPTableNumber;
+                table_nr = 0;
```

Comment:
The design was that when the VFP_TABLE was defaulted, it will use the previous value. If we set to be 0 for the defaulted one, the design will change. Please confirm this is what your testing indicates.

Reply:
I can confirm that default=0 is expected in this case. This is consistent with the manual, and also consistent with the behavior for items 9 (THP limit) and 11 (ALQ), which are both reset to 0 if defaulted.